shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/ClusterContextManagerCoordinatorTest.java

```diff
@@ -289,4 +295,53 @@ public void assertRenewInstanceLabels() {
         coordinator.renew(mockLabelsEvent);
         assertThat(contextManager.getInstanceContext().getInstance().getLabels(), is(labels));
     }
+
+    @Test
+    public void assertRenewSchemaVersionChangedEvent() {
+        when(metaDataPersistService.getDataSourceService().load("schema", "1")).thenReturn(getVersionChangedDataSourcePropertiesMap());
+        when(metaDataPersistService.getSchemaRuleService().load("schema", "1")).thenReturn(Collections.emptyList());
+        SchemaVersionChangedEvent schemaVersionChangedEvent = new SchemaVersionChangedEvent("schema", "1");
+        Map<String, DataSource> dataSourceMap = initContextManager();
+        coordinator.renew(schemaVersionChangedEvent);
+        assertThat(contextManager.getDataSourceMap("schema").get("ds_0"), is(dataSourceMap.get("ds_0")));
+        assertNotNull(contextManager.getDataSourceMap("schema").get("ds_1"));
+        assertEquals(DataSourcePropertiesCreator.create(getChangeMockedDataSource()), DataSourcePropertiesCreator.create(contextManager.getDataSourceMap("schema").get("ds_1")));
+        assertNotNull(contextManager.getDataSourceMap("schema").get("primary_ds"));
+        assertEquals(DataSourcePropertiesCreator.create(getDefaultMockedDataSource()), DataSourcePropertiesCreator.create(contextManager.getDataSourceMap("schema").get("primary_ds")));
```

Comment:
use `assertThat` to instead of `assertEquals` is better, look forward to your next pr.

Reply:
Got it. thanks